test/test_transforms_v2_refactored.py

```diff
@@ -1375,6 +1375,7 @@ def test_transform_noop(self, make_input, device):
         assert_equal(output, input)
 
 
+@pytest.mark.filterwarnings("ignore:The provided center argument has no effect")
```

Comment:
This silences:

```
(pt) ➜  vision git:(njflajnlajn) ✗ pytest test/test_transforms_v2_refactored.py -k rotate
=============================================================== test session starts ===============================================================
platform linux -- Python 3.9.12, pytest-7.0.1, pluggy-1.0.0
rootdir: /home/nicolashug/dev/vision, configfile: pytest.ini
plugins: typeguard-2.13.3, cov-3.0.0, mock-3.7.0, anyio-3.5.0
collected 3625 items / 2978 deselected / 647 selected                                                                                             

test/test_transforms_v2_refactored.py ............................................ssssssssssssssssssssssssssssssssssssssssssss............. [ 15%]
.........................................ssssssssssssssssssssssssssssssssssssssssssssssssssssss.....................ssssss................. [ 37%]
........................................................................................................................................... [ 58%]
........................................................................................................................................... [ 80%]
.................................................................................................................................           [100%]

================================================================ warnings summary =================================================================
test/test_transforms_v2_refactored.py: 96 warnings
  /home/nicolashug/dev/vision/torchvision/transforms/v2/functional/_geometry.py:982: UserWarning: The provided center argument has no effect on the result if expand is True
    warnings.warn("The provided center argument has no effect on the result if expand is True")

test/test_transforms_v2_refactored.py: 96 warnings
  /home/nicolashug/dev/vision/torchvision/transforms/v2/functional/_geometry.py:1021: UserWarning: The provided center argument has no effect on the result if expand is True
    warnings.warn("The provided center argument has no effect on the result if expand is True")

test/test_transforms_v2_refactored.py: 36 warnings
  /home/nicolashug/dev/vision/torchvision/transforms/v2/functional/_geometry.py:1037: UserWarning: The provided center argument has no effect on the result if expand is True
    warnings.warn("The provided center argument has no effect on the result if expand is True")

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html

```

Reply:
Since we have seen time and time again that in practice `center` _does_ have an effect with `expand=True` should we maybe downgrade this into a note in the documentation instead? cc @vfdev-5